SQL/0000-00-04-Help.sql

```diff
@@ -10,64 +10,11 @@ CREATE TABLE `help` (
     `parentID` int(11) NOT NULL DEFAULT '-1',
     `hash` varchar(32) DEFAULT NULL,
     `topic` varchar(100) NOT NULL DEFAULT '',
-    `content` text NOT NULL,
+    `content` text DEFAULT NULL,
```

Comment:
What is this trying to fix? I don't see why we'd want to start allowing null help content. If there's no help, it shouldn't be in the help table.
e

Reply:
When saving help content for the instrument, it got this error [Insert statement did not execute successfully: Array ( [0] => HY000 [1] => 1364 [2] => Field 'content' doesn't have a default value )].